plugins/user-authenticators/saml2/src/org/apache/cloudstack/saml/SAML2AuthManagerImpl.java

```diff
@@ -358,7 +363,8 @@ private boolean setup() {
         }
         _timer = new Timer();
         final HttpClient client = new HttpClient();
-        final String idpMetaDataUrl = SAMLIdentityProviderMetadataURL.value();
+        //final String idpMetaDataUrl = SAMLIdentityProviderMetadataURL.value();
```

Comment:
@bhaisaab  Sure. Removing the comment. The function call is needed for mocking at the time of unit testing.